README.md

```diff
@@ -260,7 +260,7 @@ To make writing queries easier there are two Grafana macros that can be used in
 
 ## Templating with Variables
 
-Instead of hard-coding things like server, application and sensor name in you metric queries you can use variables in their place. Variables are shown as dropdown select boxes at the top of the dashboard. These dropdowns make it easy to change the data being displayed in your dashboard.
+Instead of hard-coding things like server, application and sensor name in your metric queries can use variables in their place. Variables are shown as dropdown select boxes at the top of the dashboard. These dropdowns make it easy to change the data being displayed in your dashboard.
```

Comment:
The second "you" should not be deleted, i.e. "... and sensor name in your metric queries YOU can use variables..." is correct.